### ISSUE TITLE
Unable to clear errors set by updateInputsWithError on other fields or the whole form

### ISSUE THREAD

**Issue by djensen47** [formsy/formsy-react#136]
I have a login page with a username and password. The end goal is that I would like to clear the "externalErrors" when any field is updated, not just the password field.

Given a component with a Formsy form:
1. Submit the form
2. Dispatch a redux action
3. Redux logic/thunk calls the API server to login and dispatches an error
4. Props on the component are updated and `updateInputsWithError` is invoked on password with the error
5. Change the value in the password field and the error clears
6. Change the value of username

What I would like to be able to do at this point is to clear the error in the password field when the username field is changed. The only place that I found that resets the `externalError` is here:

https://github.com/formsy/formsy-react/blob/v1.1.4/src/index.js#L272-L287

That function is called in three spots in Wrapper.js
https://github.com/formsy/formsy-react/blob/master/src/Wrapper.js#L100
https://github.com/formsy/formsy-react/blob/master/src/Wrapper.js#L142
https://github.com/formsy/formsy-react/blob/master/src/Wrapper.js#L167

~I've been looking for a work around but I can't seem to figure one out.~

**Comment by djensen47** [comment]
The workaround that I found was to get a ref to my form component. The call

```javascript
this.inputRef.current.setValue(this.inputRef.current.getValue());
```

**Comment by djensen47** [comment]
Any thoughts on this from the contributors? @rkuykendall @rubencosta @MilosRasic 

Thanks.

**Comment by rkuykendall** [comment]
What about calling `reset` with the current form model? https://github.com/formsy/formsy-react/blob/master/API.md#resetvalues

**Comment by djensen47** [comment]
That could work. But it still feels like a workaround?

**Comment by rkuykendall** [comment]
Yeah I'm mixed about that as well. I don't find `reset(model)` that hackey since you _ARE_ trying to reset the validation state of the form to a 'pristine' state, but I think I've actually seen a similar request before and something like `clearValidationErrors` rings as useful to me. Was that what you were suggesting?

**Comment by djensen47** [comment]
Yeah, I think something like that would help.

**Comment by rkuykendall** [comment]
Awesome. I think this is a good issue! There's some bigger reworks I'd like to get done first but this is definitely a great idea.

**Comment by rkuykendall** [comment]
This has been fixed in the new 2.0.0 release. The new added function is `formsy.updateInputsWithValue` and it works very similarly to `updateInputsWithError`.

Please re-open this ticket or create a new one if I am mistaken.

# v2.0.0

- Much, much faster
- Many, many bug fixes
- Full Typescript typing
- Fixed deprecated lifecycle methods 
- Updated all libraries
- Fixed a number of legacy decisions in the Formsy API, mostly a reliance on function props over value props passed down to wrapped components. However, the API changes are minor and listed below:
  - `getErrorMessage()` => `errorMessage`
  - `getErrorMessages()` => `errorMessages`
  - `getValue()` => `value`
   `hasValue()` => `hasValue`
  - `isFormDisabled():` => `isFormDisabled`
  - `isFormSubmitted()` => `isFormSubmitted`
  - `isPristine()` => `isPristine`
  - `isRequired()` => `isRequired`
  - `isValid():` => `isValid`
  - `showError()` => `showError`
  - `showRequired()` => `showRequired`

https://github.com/formsy/formsy-react/compare/v1.1.4...v2.0.0
